shakedown/dcos/marathon.py

```diff
@@ -0,0 +1,22 @@
+from dcos import marathon
+from shakedown.dcos.spinner import *
+
+
+def deployment_predicate(client=None):
+    if client is None:
+        client = marathon.create_client()
+    return len(client.get_deployments()) == 0
+
+
+def deployment_wait(timeout=120):
+    time_wait(deployment_predicate, timeout)
```

Comment:
I should remove the passed in client on the predicate.   I thought an end user may pass one.. however after thinking about it,  it isn't necessary.  the marathon.create_client will give them whatever marathon is configured by default it will be the root marathon.   if they use the marathon_on_marathon context they will get another marathon.   I plan to move the marathon_on_marathon context on another pr.   that said there are other ways to configure the expected marathon external to the method call.